src/components/endpoints/Assets.vue

```diff
@@ -64,6 +65,8 @@ const create = async () => {
   const erc725AccountAddress = getState('address')
   isTokenPending.value = true
 
+  const assetMetadata = await uploadAssetData(lsp4Metadata.value)
```

Comment:
Even though uploads seem to be successful for some reason the images are not loading 🤔 

A response uploading one of the images for LSP7:
```
{
    "PinSize": 530,
    "Timestamp": "2024-05-29T21:53:06.161Z",
    "IpfsHash": "QmXTe24kghccU8DK8qobjuQXDqMwQWCJQPAaZuorBWMvA9",
    "hash": "QmXTe24kghccU8DK8qobjuQXDqMwQWCJQPAaZuorBWMvA9"
}
```